### PR TITLE
Remove SQL data management url and add link to insert

### PR DESCRIPTION
**Pull request by rosewms**
## Description

* Remove url for Data management
* Add a specific url for INSERT

> Is this change a fix, improvement, new feature, refactoring, or other? Fix/improvement

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific) Memory connector

> How would you describe this change to a non-technical end user or system administrator? Specifying which data management commands are supported. 

## Related issues, pull requests, and links

* Error was reported in the Slack troubleshooting channel, here's the [thread](https://trinodb.slack.com/archives/CGB0QHWSW/p1646997954848359) with the info.

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

### CODE REVIEW

File: docs/src/main/sphinx/connector/memory.rst [hunk 1]
```diff
@@ -51,12 +51,13 @@ stored in memory. In addition to the :ref:`globally available
...
 statements, the connector supports the following features:
 
-* :ref:`sql-data-management`
 * :doc:`/sql/create-table`
 * :doc:`/sql/create-table-as`
 * :doc:`/sql/drop-table`
 * :doc:`/sql/create-schema`
 * :doc:`/sql/drop-schema`
+* :doc:`/sql/insert`
+* :doc:`/sql/update`
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/memory.rst:60]: Move to the top of the list, so insert > update > create-table.

**Review by jhlodin — APPROVED** [review]
LGTM % ordering nit

**Comment by mosabua** [comment]
The real reason is the update is not supported... but that is in the list for data management.

**Review by mosabua — APPROVED** [review]

File: docs/src/main/sphinx/connector/memory.rst [hunk 2]
```diff
@@ -51,7 +51,8 @@ stored in memory. In addition to the :ref:`globally available
 <sql-globally-available>` and :ref:`read operation <sql-read-operations>`
 statements, the connector supports the following features:
 
-* :ref:`sql-data-management`
+* :doc:`/sql/insert`
+* :doc:`/sql/update`
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/memory.rst:55]: Please remove this line as the connector doesn't support `UPDATE` statement.

**Review by ebyhr — APPROVED** [review]
Looks good except for a `UPDATE`. Please update the commit & PR title when you address the comment.

**Comment by rosewms** [comment]
@ebyhr thank you for your help, I made the requested changes so this PR should be ready to merge.

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**